framework/test-vectors/encryption-manifest.md

```diff
@@ -0,0 +1,153 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Encryption Manifest
+
+## Version
+
+1.0.0
+
+## Dependencies
+
+This serves as a reference of all features that this feature depends on.
+
+| Feature                                         | Min Version | Max Version |
+| ----------------------------------------------- | ----------- | ----------- |
+| [keys-description](./key-description.md)        | 1           | n/a         |
+| [keys-manifest](./keys-manifest.md)             | 1           | n/a         |
+| [encryption-manifest](./encryption-manifest.md) | 1           | n/a         |
+
+## Summary
```

Comment:
I think we need some words somewhere to better explain why the encryption-manifest is distinct from the decryption-manifest.